.travis.yml

```diff
@@ -5,7 +5,9 @@ matrix:
       install:
         - env | grep TRAVIS > .env
         - env | grep CI >> .env
-      script: docker-compose -f docker-compose.yml -f docker-compose.travis.yml run web tox
+        - echo CELERY_TASK_ALWAYS_EAGER=True >> .env
+        - echo SECRET_KEY=not_very_secret_key >> .env
```

Comment:
This feels a bit weird. Would it be better to (a) set these in `docker-compose.travis.yml`, or (b) add a new file for travis-specific env vars and `cat` that into `.env` here instead of using individual `echo`s?